docker/mssql-techempower/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM microsoft/mssql-server-linux:2017-latest
```

Comment:
I think we should use `2017-GA` instead of `2017-latest`.  It's generally better to use specific versions for consistency and test perf when upgrading.  Can you also please change `postgres:9` to `postgres:9.6.5`?